scripts/check_package_json.ts

```diff
@@ -0,0 +1,30 @@
+import { glob } from 'glob';
+import { readPackageJson } from './components/package-json/package_json.js';
+
+const packagePaths = await glob('./packages/*');
+
+const tempExclude: string[] = [
+  '@aws-amplify/plugin-types',
+  '@aws-amplify/cli-core',
+];
```

Comment:
```suggestion

// Excluding packages that violate 'import/no-extraneous-dependencies' rule until they're fixed.
const tempExclude: string[] = [
  '@aws-amplify/plugin-types',
  '@aws-amplify/cli-core',
];
```

Please also cut GH issue to follow up on this (and perhaps include link to it in the comment).

Reply:
issue created: https://github.com/aws-amplify/amplify-backend/issues/2001